deisctl/client/client.go

```diff
@@ -120,13 +121,25 @@ After install, the components will be available to start.
 
 Usage:
   deisctl install [<target>...] [options]
-`
+
+Options:
+  --router-mesh-size=<num>  Number of routers to be loaded when installing the platform [default: %d].
+`, cmd.DefaultRouterMeshSize)
 	// parse command-line arguments
 	args, err := docopt.Parse(usage, argv, true, "", false)
 	if err != nil {
 		return err
 	}
 
+	meshSizeArg, ok := args["--router-mesh-size"].(string)
+	if ok {
```

Comment:
I know that this will likely never happen, but I still think we should have a catch case in case `--router-mesh-size` comes up blank, or else we should just remove this `if ok` branch logic and just go about our business.